assets/javascript/app.js

```diff
@@ -1,190 +1,200 @@
-//Array of questions that are also objects
 var questions = [{
-        questionNum: "question-0",
+        questionNum: 0,
         question: 'How many Stark children are there at the start of the show?',
         choices: [4, 5, 6, 7],
-        answer: "choice-1"
+        answer: 1
     },
     {
-        questionNum: "question-1",
+        questionNum: 1,
         question: 'How many children does Cersei have at the start of the show?',
         choices: [0, 2, 4, 3],
-        answer: "choice-3"
+        answer: 3
     },
     {
-        questionNum: "question-2",
+        questionNum: 2,
         question: 'What continent is the seven kingdoms on?',
         choices: ["Easteros", "Planetos", "Westeros", "Asshai"],
-        answer: "choice-2"
+        answer: 2
     },
     {
-        questionNum: "question-3",
+        questionNum: 3,
         question: 'Another name for Oberyn Martell is?',
         choices: ["the viper", "the lion", "the shark", "the cobra"],
-        answer: "choice-0"
+        answer: 0
     },
     {
-        questionNum: "question-4",
+        questionNum: 4,
         question: 'How many direwolf pups are there at the start of the show',
         choices: [4, 5, 6, 7],
-        answer: "choice-2"
+        answer: 2
     },
     {
-        questionNum: "question-5",
+        questionNum: 5,
         question: 'What was the Mad King\'s first name',
         choices: ["Aemon", "Aerys", "Aegon", "Armon"],
-        answer: "choice-1"
+        answer: 1
     },
     {
-        questionNum: "question-6",
+        questionNum: 6,
         question: 'What is the name of the leader of the dead',
         choices: ["Winter King", "Nightmare", "Night King", "Frosty the Snowman"],
-        answer: "choice-2"
+        answer: 2
     },
     {
-        questionNum: "question-7",
+        questionNum: 7,
         question: 'What is Arya\'s sword\'s name',
         choices: ["Needler", "Needle", "Pointy", "Mr. Stick"],
-        answer: "choice-1"
+        answer: 1
     },
     {
-        questionNum: "question-8",
+        questionNum: 8,
         question: 'What are the Greyjoy\'s words',
         choices: ["We Do Not Bow", "Family,Duty,Honor", "We Do Not Sow", "Winter is Coming"],
-        answer: "choice-2"
+        answer: 2
     },
     {
-        questionNum: "question-9",
+        questionNum: 9,
         question: 'What is Jon Snow\'s sword name ',
         choices: ["Longclaw", "Sharpclaw", "Longpaw", "Bastardclaw"],
-        answer: "choice-0"
+        answer: 0
     }
 ];
 
 var correctGuesses = [];
 var wrongGuesses = [];
 var intervalId;
 
+var audio = document.getElementById("myAudio");
```

Comment:
highly recommend naming your variables explicitly. When you come back and look at your code from a long time ago, it'll be harder for you to understand what `x` is just by looking at it. Also difficult for others to contribute to your code.